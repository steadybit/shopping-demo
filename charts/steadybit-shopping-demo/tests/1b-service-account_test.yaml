templates:
  - 1b-service-account.yml
chart:
  appVersion: v0.0.0
tests:
  - it: manifest should match snapshot
    set:
    asserts:
      - matchSnapshot: { }