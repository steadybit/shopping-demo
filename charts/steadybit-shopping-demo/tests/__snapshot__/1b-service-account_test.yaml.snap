manifest should match snapshot:
  1: |
    apiVersion: v1
    items:
      - apiVersion: v1
        kind: ServiceAccount
        metadata:
          name: steadybit-demo
          namespace: NAMESPACE
      - apiVersion: rbac.authorization.k8s.io/v1
        kind: RoleBinding
        metadata:
          name: steadybit-demo:view
          namespace: NAMESPACE
        roleRef:
          apiGroup: rbac.authorization.k8s.io
          kind: ClusterRole
          name: view
        subjects:
          - apiGroup: rbac.authorization.k8s.io
            kind: Group
            name: system:serviceaccounts:steadybit-demo
    kind: List
